identifiers/country-kr/cities.csv

```diff
@@ -0,0 +1,9 @@
+id,name

```

Comment:
Please sort this file

Reply:
It was sorted by `id`. I have now sorted it by `name`.

Reply:
I think `id` is the correct column to sort by, but `ocd-division/country:kr/metropolitan_city:31` was at the end out of order. Could you resort by `id`?

Reply:
Done